docs/UserGuide.md

```diff
@@ -557,21 +626,24 @@ Examples:
 * `paid 1 1 amt/70` The 1st student has paid `$70.00` for his or her 1st lesson.
 * `paid 3 2 amt/480.50` The 3rd student has paid `$480.50` for his or her 2nd lesson.
 
-#### Behaviours of the Fees Calculator
+#### Behaviour of the Fees Calculator
 
 TAB will automatically update your lesson's outstanding fees once the lesson has ended using Fees Calculator feature. 
 The Fees Calculator will account for cancelled dates and ensure that lesson fees on these dates will not be added.
 
-However, the Fees Calculator will not account for any changes to lessons that have passed. Such cases include:
+<div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
+The Fees Calculator will not account for any changes to lessons that have passed.
+</div>
 
+Such cases to take note of include:
 * **Lesson rates increment.** In the event that you want to increase your lesson rates, the current outstanding fees will not change according to the newly edited lesson rates.
-* **Incorrect lesson rates entry.** Similarly, in the event that you have entered your lesson rates incorrectly and only realised it after your lesson has passed, the current outstanding fees will not change according to 
-the newly edited lesson rates.
+* **Incorrect lesson rates entry.** Similarly, in the event that you have entered your lesson rates incorrectly and only realised it after your lesson has passed, the current outstanding fees will not change according to
```

Comment:
Why do you need this point? It feels a bit repetitive... How about just combine the two points and make it more general like
`Changing of Lesson rates` -> The current outstanding fees will not change according to the newly edited lesson rates.